<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберите сторону:</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radBut_1">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radBut_2">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pBut_fire">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>651</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбор города для атаки:</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QComboBox" name="comBox_iran"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comBox_izrail"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>651</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Ход операции: </string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="3">
      <widget class="QProgressBar" name="pBar_ashdod">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Исфахан</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QProgressBar" name="pBar_icfahan">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QProgressBar" name="pBar_ierusalim">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QProgressBar" name="pBar_keradj">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QProgressBar" name="pBar_telaviv">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Хайфа</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="pBar_tegeran">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Иерусалим</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Мешхед</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Тегеран</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Ашдод</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="pBar_meshhed">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Кередж</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QProgressBar" name="pBar_haifa">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Тель-Авив</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QProgressBar" name="pBar_Rishon">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QProgressBar" name="pBar_shiraz">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Шираз</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Ришон-ле-Цион</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
